bodhi/server/templates/update.html

```diff
@@ -685,19 +775,14 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
-              % if request.registry.settings.get('ci.required'):
+              % if request.registry.settings.get('greenwave_enabled'):
               <div class="p-b-1">
                 <div>
-                  % if request.registry.settings.get('ci.url'):
-                    <a href="${request.registry.settings.get('ci.url')}" title="More information about CI">
```

Comment:
I guess the value of ci.url would be the ci centos, right? I do not think it makes sense to show it if we decide to change it to 'Gating Status'.